packages/esm-patient-outcomes-app/src/smsform/send-sms-form.component.tsx

```diff
@@ -77,6 +94,7 @@ const SendSmsForm: React.FC<SendSmsFormProps> = ({
         body: body,
         source: source,
         patientUuid: patientUuid,
+        locale: locale,
```

Comment:
There are a few typos below this line that aren't part of this PR, but should be fixed.